tasks/createCache.js

```diff
@@ -0,0 +1,108 @@
+const fs = require('fs');
+const path = require('path');
+
+let packageJSON = require('../package.json');
+
+const root = path.join(__dirname, '..');
+
+/**
+ * @param { String } v
+ * @returns { String } bumped version
+ */
+function versionBump(v) {
+  v = v.split('.');
+  v[2]++;
+  return v.join('.');
+}
+
+const version = versionBump(packageJSON.version);
+
+packageJSON.version = version;
+packageJSON = JSON.stringify(packageJSON, null, 2);
+
+/**
+ * @param { string } dir
+ * @param { Array } filelist
+ * @returns { Array } all html files, all.css and all.bundle.js
+ */
+const walkSync = (dir, filelist = []) => {
```

Comment:
Could we replace this with [glob](https://npmjs.org/package/glob)? That would be much more readable.

Reply:
It bumps package.json version coz cache version is taken from there.